iOS/Data/Model/CourseItem+share.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Common
+import LinkPresentation
+import UIKit
+
+extension CourseItem: UIActivityItemSource {
+
+    var combinedTitle: String {
+        return [self.title, self.section?.course?.title].compactMap { $0 }.joined(separator: " - ")
+    }
+
+    public func activityViewControllerPlaceholderItem(_ activityViewController: UIActivityViewController) -> Any {
+        return self.combinedTitle
```

Comment:
For sharing courses, we are returning the URL. Which is the correct value to return here?

Reply:
I think using the combined title is a good choice :)